testdata/virtual_wan_health_check.jsonnet

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
Please add a comment as the first line what URL you used to get the API dump so we can refresh or compare it if needed. See the other jsonnet files for examples.